src/coerce-functions.js

```diff
@@ -0,0 +1,51 @@
+import * as LogManager from 'aurelia-logging';
+
+export const coerceFunctions = {
+  none(a) {
+    return a;
+  },
+  number(a) {
+    const val = Number(a);
+    return !isNaN(val) && isFinite(val) ? val : 0;
+  },
+  string(a) {
+    return '' + a;
+  },
+  boolean(a) {
+    return !!a;
+  },
+  date(val) {

```

Comment:
Since dates are stateful, should there be a check if it's already of type date?
I haven't thought this through, just want to make sure this case has been considered.
```
if (typeof val === 'date') {
  return val;
}
```

Reply:
This could be nice. I didn't think of it. Maybe we can reuse the instance ? @jdanyow / @EisenbergEffect